.scripts/bump_script_version.py

```
@@ -0,0 +1,105 @@
+            has_script_changes = True
+            break
+
+    if not has_script_changes:
+        print("No changes to test script detected.")
+        sys.exit(0)
+
+    # 2. Check if version was already bumped
+    def get_version_at(ref):
+        try:
+            content = run_command(["git", "show", f"{ref}:{config_path}"])
+            if not content: return None
```

Ruff (0.14.10)

Multiple statements on one line (colon)

(E701)